ci/cloudbuild/build.sh

```diff
@@ -313,6 +315,7 @@ subs+=("_BUILD_NAME=${BUILD_NAME}")
 subs+=("_TRIGGER_SOURCE=manual-${account}")
 subs+=("_PR_NUMBER=") # Must be empty or a number, and this is not a PR
 subs+=("_LOGS_BUCKET=${project}_cloudbuild")
+subs+=("_LOG_FORMAT=$([[ -t 0 ]] && echo color || echo text)")
```

Comment:
This doesn't look quite right to me. It looks like this is checking if the _local_ script is running with a tty, and based on that it's telling the _remote_ GCB build whether or not to use color.

Additionally, for triggered builds, this part of the script won't be run at all. A triggered build enters this script with the `--local` flag. So my guess is that in `cloudbuild.yaml`, kaniko is being run with `--log-format=""`, which seems like it uses `text` in that case?

If this is correct, maybe we should just change the `cloudbuild.yaml` file to unconditionally pass `--log-format=text` for GCB builds, which I think, will never have a tty. WDYT?

Reply:
I'm happy to go with the unconditional `--log-format=text`, so I'll update shortly.

That said, I can get both `text` and `color` behavior respectively with, say, ...
```
ci/cloudbuild/build.sh --trigger=asan-pr </dev/null 
ci/cloudbuild/build.sh --trigger=asan-pr
```

I don't think we'd ever generate `--log-format=""`.

Reply:
_We_ wouldn't generate `--log-format=""`, but GCB would when it triggers a PR/CI build because it doesn't use `build.sh` to start that build. So the `_LOG_FORMAT` substitution would be unset and would expand to the empty string when referenced in `cloudbuild.yaml`.

It is a bit confusing because `build.sh --trigger <file>` just means to look up the `_DISTRO` and `_BUILD_NAME` from the named trigger file, then submit the build to GCB. That's different than a build that's automatically triggered for a PR or CI build.

Regardless, I _think_ GCB will never run our builds with a TTY, so we might as well just tell Kaniko to use text logs unconditionally.